system-x/services/db/common/src/main/java/software/tnb/db/common/openshift/OpenshiftDB.java

```diff
@@ -29,6 +32,19 @@ public OpenshiftDB(SQL sqlService, int port) {
 
     @Override
     public void create() {
+        SecurityContextConstraints scc = OpenshiftClient.get().securityContextConstraints().withName(SCC_NAME).get();
+        if (scc == null) {
+            SecurityContextConstraints restricted = OpenshiftClient.get().securityContextConstraints().withName("restricted").get();
+            scc = OpenshiftClient.get().securityContextConstraints().create(
+                new SecurityContextConstraintsBuilder(restricted)
+                    .withNewMetadata() // new metadata to override the existing annotations
```

Comment:
shouldn't the original metadata be saved so that they can be restored in `undeploy`?

Reply:
I'm not sure why - the new SCC we create here AFAIK only requires the new metadata to change the SCC name - so we can use it also on OSD clusters. The original metadata is still stored in with the `restricted` SCC.